docker-compose.yml

```diff
@@ -23,10 +23,14 @@ services:
   jellyfish-tests:
     build:
       context: .
-      dockerfile: Dockerfile.template
+      dockerfile: Dockerfile.jf-tests
```

Comment:
I think you can leave the filename as `Dockerfile.template` as we're not using it anywhere else. `Dockerfile.template` also has a use/meaning in the balena world.

Reply:
Ok, will revert.

Reply:
Also check comment below